### PR TITLE
Avoid duplicate import in IPEX auto importer

### PR DESCRIPTION
**Pull request by qiyuangong**
### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: python/llm/src/ipex_llm/utils/ipex_importer.py [hunk 1]
```diff
@@ -63,10 +74,11 @@ def import_ipex(self):
             if 'ipex' in sys.modules or 'intel_extension_for_pytorch' in sys.modules:
                 logging.error("ipex_llm will automatically import intel_extension_for_pytorch.")
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/utils/ipex_importer.py:75]: The error message should be "Please import ipex_llm before importing intel_extension_for_pytorch"

File: python/llm/src/ipex_llm/utils/ipex_importer.py [hunk 2]
```diff
@@ -63,10 +74,11 @@ def import_ipex(self):
...
                 log4Error.invalidInputError(False,
-                                            "Please import ipex_llm before importing \
-                                                intel_extension_for_pytorch!")
+                                            "Please don't import intel_extension_for_pytorch manually!")
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/utils/ipex_importer.py:77]: The error message should be "Please import ipex_llm before importing intel_extension_for_pytorch"

File: python/llm/src/ipex_llm/utils/ipex_importer.py [hunk 3]
```diff
@@ -19,6 +19,37 @@
...
+    # check import calling pacage
+    calling_package = get_calling_package()
+    if calling_package is not None:
+        print("origin import")
+        return original_import(name, globals, locals, fromlist, level)
+    # Only check ipex for main thread
+    if name == "ipex" or name == "intel_extension_for_pytorch":
+        logging.error("intel_extension_for_pytorch has already been automatically imported.")
+        log4Error.invalidInputError(False,
+                                    "Please avoid importing it again!")
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/utils/ipex_importer.py:51]: Do not split the error message

File: python/llm/src/ipex_llm/utils/ipex_importer.py [hunk 4]
```diff
@@ -61,12 +92,13 @@ def import_ipex(self):
         if self.is_xpu_version_installed():
             # Check if user import ipex manually
             if 'ipex' in sys.modules or 'intel_extension_for_pytorch' in sys.modules:
-                logging.error("ipex_llm will automatically import intel_extension_for_pytorch.")
+                logging.error("intel_extension_for_pytorch has already been automatically imported.")
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/utils/ipex_importer.py:95]: This error message is wrong

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**